Dvorak-Right-Handed/KeyboardViewController.swift

```diff
@@ -127,7 +135,7 @@ class KeyboardViewController: UIInputViewController {
                 switch title {
                 case "⌫" :
                     proxy.deleteBackward()
-                case "⏎" :
+                case "enter" :
```

Comment:
Match above if changed to "Enter" or "ENTER". Consider Constants to replace using quoted string text in more than one place as a way to prevent bugs from not updating all instances of the quoted text.